lib/LTP/WhiteList.pm

```diff
@@ -19,23 +18,28 @@ use Mojo::UserAgent;
 use Mojo::File 'path';
 use YAML::PP;
 
-our @EXPORT = qw(
-  download_whitelist
-  find_whitelist_testsuite
-  find_whitelist_entry
-  override_known_failures
-  is_test_disabled
-  list_skipped_tests
-);
+sub new {
+    my $class = shift;
+    my $self = bless({}, $class);
+    my $path = get_var('LTP_KNOWN_ISSUES_LOCAL');
```

Comment:
I wonder what the point to create this class when this variable is not defined ? Shouldn't we simply fail here ?

Reply:
No, we should not fail here. Running LTP tests without any whitelist file is perfectly valid.

Reply:
Also, if `LTP_KNOWN_ISSUES_LOCAL` is not defined, it just means that this is the first ever class instance in the job and the file needs to be downloaded.